components/policy-editor/org.wso2.carbon.policyeditor.ui/src/main/resources/web/policyeditor/css/local-styles.css

```diff
@@ -1,16 +1,19 @@
-/* Copyright 2006-2008 WSO2, Inc. http://www.wso2.org
+/*
+ * Copyright (c) 2008, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This file has been created and added in 2008, should I change year? if so what should I use?